apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -287,10 +315,12 @@ get_top_non_empty_chain(_Config) ->
 
 block_by_height(_Config) ->
     BlocksToCheck = 4,
+    InitialHeight = aec_blocks:height(rpc(aec_conductor, top, [])),
+    BlocksToMine = max(BlocksToCheck - InitialHeight, 0),
     aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE),
-                                   BlocksToCheck),
+                                   BlocksToMine),
     TopHeader = rpc(aec_conductor, top_header, []), 
-    BlocksToCheck = aec_headers:height(TopHeader),
+    true = BlocksToCheck =< aec_headers:height(TopHeader),
```

Comment:
It would be nice to show bad height in case of error in order to have more clues on the failure.
E.g. `case aec_headers:height(TopHeader) of TopHeaderHeight when TopHeaderHeight >= BlocksToCheck end,`